src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -6,6 +6,12 @@
  */
 public class Bulldozer extends Machine {
     public Bulldozer() {
+        super();
+    }
+
+    public Bulldozer(String color, String name) {
+        this.color = color;
+        this.name = name;
```

Comment:
```suggestion
        super(color, name);
```
better to create constructor in parent class, because these fields are in class `Machine` and should be reached through parent class.